enhancements/single-node-developer-cluster-profile.md

```diff
@@ -0,0 +1,514 @@
+```
+
+This annotation will need to be added to the manifests for all the
+OpenShift components that are not currently removed by the snc.sh
+script during the CRC build process. The components that are currently
+removed, and thus should not need the annotation added, include:
+
+* most operators running in the openshift-monitoring namespace
+* most operators running in the openshift-insights namespace
+* machine-api-operator
+* Machine-config-operator (may need to run once, requiring the annotation)
+* openshift-cloud-credential-operator
+* openshift-cluster-storage-operator
+
+Additional components that are not currently removed by the SNC
+tooling might be also be determined to be unnecessary, and thus not
+require the annotation. Possibilities include:
+
+* openshift-cluster-node-tuning-operator
+* openshift-multus
+
+Once PRs adding this annotation have been merged to all the necessary
+component repositories, the snc.sh script will be updated to tell the
+OpenShift installer to use the single-node-developer cluster profile,
+and to remove the code that deletes the components that are not
+annotated. At the conclusion of this initial phase, the somewhat
+simplified CRC build tooling should produce a cluster equivalent to
+that currently produced, and all existing CRC tests should pass. Phase
+one is expected to be completed during development of the 4.7
+OpenShift release.
+
+From this point forward, any new components being added to OpenShift
+will not be included in CRC unless a manifest with the
+single-node-developer annotation is included in the new component. The
+CRC team will need to engage with other OpenShift teams to help decide
+whether or not new components should be included in the profile, and
+if so, how they should be configured.
+
+### Phase Two
+
+In the second phase, various OpenShift components that are included in
+single node developer clusters will be updated to use specialized
+manifests in the single-node-developer profile rather than the same
+manifest used for default clusters. These specialized manifests will
+configure the components for use in single node developer clusters,
+replacing the patching currently done post-deployment by the CRC build
+process, and potentially enabling additional reductions in resource
+requirements.
+
+Some of the manifest specializations expected to be included in this
+phase are shown in the following table:
+
+
+| Operator | Change |
+| :--- | :--- |
+| ingress | reduce the number of replicas of controller pod |
+| console | reduce the number of replicas of console pod |
+| console | reduce the number of replicas of download pod |
+| authentication | reduce the number of replicas of oauth pod |
+| lifecycle-manager | reduce the number of replicas of packageserver pod |
+| etcd | have a configuration with useUnsupportedUnsafeNonHANonProductionUnstableEtcd option |
+| monitoring | disable it but still create the CRDs (esp. PrometheusRule and ServiceMonitor) |
```

Comment:
existing crc had instructions on how to enable function that was disabled in your existing flow.
are you intending to provide a mechanism to enable monitoring (or the parts you disable)?